### ISSUE TITLE
Provide a clean dark theme by default

### ISSUE THREAD

**Issue by echebbi** [eclipse/xtext-xtend#1078]
## Motivation

Default colors used by the Xtend editor make the source code not really readable:

![image](https://user-images.githubusercontent.com/25926433/85926668-3c5a0780-b8a1-11ea-969d-bb9ef4d5b8f0.png)

## Proposed solution

Override default CSS preferences when Eclipse's theme is dark. I personally use a colorful theme with JDT's colors:

![image](https://user-images.githubusercontent.com/25926433/85926711-8642ed80-b8a1-11ea-8862-7a68643f2f6e.png)

**Comment by szarnekow** [comment]
Would you be willing to contribute a patch towards your proposed solution. That would be greatly appreciated.

**Comment by echebbi** [comment]
I'd be glad to. Should I first propose my own color theme?

**Comment by szarnekow** [comment]
Yes, I think that's a good idea since it closely resembles JDTs colors.

**Comment by cdietrich** [comment]
Move to 2.24

**Comment by kthoms** [comment]
![image](https://user-images.githubusercontent.com/265597/99800956-820bbb00-2b35-11eb-8fbc-7c19d6b5db86.png)

**Comment by kthoms** [comment]
@echebbi We have fixed this now with inspiration from you initial request. Thanks for bringing this up!

**Comment by cdietrich** [comment]
needs to be reworked due to regressions
